README.md

```diff
@@ -44,15 +44,33 @@ You can connect to the Playground using the JavaScript client. Here's an example
 			}
 		</script>
 		<script type="module">
-			import { connectPlayground, login } from '@wp-playground/client';
-
-			const client = await connectPlayground(
-				document.getElementById('wp'),
-				{ loadRemote: 'https://playground.wordpress.net/remote.html' }
-			);
-			await client.isReady();
-			await login(client, 'admin', 'password');
-			await client.goTo('/wp-admin/post-new.php');
+			import { startPlayground } from '@wp-playground/client';
+
+			const client = await loadPlayground({
```

Comment:
From the import statement above, and what the `playground/client` package exports ([here](https://github.com/WordPress/wordpress-playground/blob/trunk/packages/playground/client/src/index.ts#L33)), this line is probably meant to call `startPlayground` instead of `loadPlayground` (also mentioned in the pull request title)?

Reply:
Great catch, thank you!